plutarch-test/plutarch-test.cabal

```diff
@@ -87,6 +89,7 @@ common deps
     , serialise
     , sydtest
     , sydtest-discover
+    , sydtest-hedgehog
```

Comment:
@L-as Are you okay with this? hedgehog or quickcheck?

Reply:
Just a little addition: geometer used the `PropertyT` transformer which is not in quickcheck.

Reply:
I honestly prefer quickcheck but :shrug: I wanted to merge this so other people could work on it